### DISCUSSION TITLE
New Nicehash Algo

### DISCUSSION THREAD

**Discussion by acos0874** [RainbowMiner/RainbowMiner#1677] [Q&A] [answered]
Just wanted to let you know Autolykos just got added to nicehash

**Comment by RainbowMiner** [comment]
Thank you! I'll check the API.

**Comment by RainbowMiner** [comment, marked as answer]
Ok, it's arriving via their API correctly - but no rentals, currently. That's why it isn't listed in our pools-list, at the moment. As soon, as there are buyers, it will be enabled by RainbowMiner automatically.

**Comment by acos0874** [comment]
Thank you!

**Comment by acos0874** [comment]
Some errors being reported when trying to mine it.

**Comment by mo35** [comment]
i narrowed down issue with pool urls , when bench failed , it was trying to mine to eu/us/hk such locations , there is no such things for Ergo,  example from log :
\GPU-NBMiner\nbminer.exe --api 127.0.0.1:34000 -o stratum+tcp://autolykos.**eu**.nicehash.com:3390 -u xxx

from NH page 
Autolykos
stratum+tcp://autolykos.LOCATION.nicehash.com:3390

If you use 3rd party software, connect directly to our stratum server
LOCATION: eu-west, eu-north, usa-west, usa-east

i tested with standalone nbminer with proper location , it works fine

**Comment by RainbowMiner** [comment]
Oh dear! Yes, it seems, that Nicehash has changed their servers. I will add a fix asap.

**Comment by RainbowMiner** [comment]
Ok, it looks like, that Nicehash's international routing proxies aren't working for Autolykos, currently. Fix is there.
